src/main/java/com/adaptris/filesystem/smbj/SMBConsumer.java

```diff
@@ -90,7 +88,7 @@
   /**
    * Set the filename filter implementation that will be used for filtering files.
    * <p>
-   * The expression that is used to filter messages is obtained from the associated {@link ConsumeDestination}, if not specified,
+   * The expression that is used to filter messages, if not specified,
```

Comment:
Should be "derived from #getFilterExpression"  rather than just deleting it. The sentence doesn't make sense anymore